assets/javascripts/discourse/initializers/extend-for-solved-button.js.es6

```diff
@@ -155,6 +161,46 @@ function initializeWithApi(api) {
     }
   });
 
+  if (mobileView) {
+    api.decorateWidget("post-contents:after-cooked", dec => {
+      const model = dec.getModel();
+      const result = [];
+      if (model.accepted_answer) {
+        result.push(
+          dec.attach("button", {
+            label: "solved.solution",
+            title: "solved.solution",
+            icon: "check-circle",
+            action: "noop",
```

Comment:
Is that needed?

Reply:
Yes, it is needed. Otherwise, it will try to access the `undefined` action.